source/spp/mnm/spp_117_nl_abiotiek_waterlopen/03_benodigdheden.Rmd

```diff
@@ -0,0 +1,33 @@
+
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| Algemene benodigdheden                                              | Specifieke benodigdheden deelprotocollen                          |
++=====================================================================+===================================================================+
+| -   Staalnameformulier (analoog of digitaal)                        | -   Stroomsnelheidsmeter                                          |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Klembord, potlood en slijper                                    | -   Horizontale waterhapper, voorzien van een voldoende lang touw |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Laarzen, lieslaarzen, waadpak (site-afhankelijk)                | -   Telescopische steel met maatbeker                             |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Latex handschoenen (poederloos)                                 |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Plooimeter                                                      |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Fototoestel                                                     |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+| -   Documentatie van het staalnamepunt om dit te kunnen lokaliseren |                                                                   |
++---------------------------------------------------------------------+-------------------------------------------------------------------+
+
+: (#tab:Tabel1) Checklist benodigd veldmateriaal.
+
+## Materiaal
+
+Zie materiaal van de deelprotocollen ([sfp-113-nl 2023.04](../2023.04/index.html), [sfp-114-nl 2024.03](../2024.03/index.html), [sfp-115-nl 2024.04](../2024.04/index.html)), aangevuld met:
+
+-   Horizontale waterhapper (bijvoorbeeld Friedinger, Van Dorn, …) voorzien van nylonkoord of touw.
+
+-   Polyethyleen of polypropyleen maatbeker van 2 liter (+ reserve) voorzien van gietmondje en handvat.
```

Comment:
dit mag geschrapt worden; dit staat wellicht al vermeld in sfp-114

Reply:
Is nu in orde